stablehlo/reference/Ops.cpp

```diff
@@ -56,6 +56,14 @@ Tensor eval(FloorOp op, const Tensor &operand) {
   return result;
 }
 
+Tensor eval(NegOp op, const Tensor &operand) {
+  Tensor result(op.getType());
+  for (auto it = operand.index_begin(); it != operand.index_end(); ++it) {
+    result.set(*it, -operand.get(*it));
+  }
+  return result;
+}
```

Comment:
Drive by: seems like we're creating a new tensor unconditionally, but @sdasgup3 was telling me yesterday that Tensor is mutable. 
Seems like we could check if we're the only (or better: last) uses of the operand and mutate it in-place.